corehq/apps/reports/standard/monitoring.py

```diff
@@ -1438,8 +1438,13 @@ def headers(self):
 
     @property
     def group_ids(self):
-        return [group_id for group_id in self.request.GET.getlist('group')
-                if group_id and group_id != '_all']
+        from corehq.apps.userreports.reports.filters.values import CHOICE_DELIMITER
+        group_ids = []
+        for group_id in self.request.GET.getlist('group'):
+            if group_id and group_id != '_all':
+                # ReportConfig objects saved from JS could have this delimiter
```

Comment:
The ambiguity of this comment isn't a great sign. Do you know if it's possible to fix this on the js side instead, probably by alteering https://github.com/dimagi/commcare-hq/commit/8e086c1a752f84a1e87a83f7eac496273fbc54ab#diff-c185b3f356890af823a32078b0c7c0e9R255 ?

Reply:
Ya. better to fix it at the source. if this fix is needed urgently, then fine to merge this and follow up with JS fix.

Reply:
So, I considered that, but it seems as if it's an established strategy (for example, if you grep `CHOICE_DELIMITER` you will find more instances) and there might already be `ReportConfig` docs lying with these chars. It's not easy to scrape through ReportConfig objects that have this char in the filter and fix those.

The `could` is because this happens only when multiple choices are picked. Happy to consider reframing.

Reply:
Actually please see my latest comment on https://dimagi-dev.atlassian.net/browse/HI-730